.github/scripts/trymerge.py

```diff
@@ -591,7 +627,13 @@ def find_matching_merge_rule(pr: GitHubPR, repo: GitRepo, force: bool = False) -
     reject_reason_score = 0
     for rule in rules:
         rule_name = rule.name
-        rule_approvers_set = set(rule.approved_by)
+        rule_approvers_set = set()
+        for approver in rule.approved_by:
+            if "/" in approver:
```

Comment:
are all teams org/name format?

Reply:
They should be

Reply:
Yes they are, according to https://github.blog/2012-05-09-introducing-team-mentions/